packages/design-system-react/src/components/blockies/Blockies.stories.tsx

```diff
@@ -13,21 +13,31 @@ const meta: Meta<BlockiesProps> = {
       page: README,
     },
   },
+  args: {
+    address: '0x9Cbf7c41B7787F6c621115010D3B044029FE2Ce8',
+    size: 32,
+  },
```

Comment:
Displays defaults in controls

![Screenshot 2025-03-18 at 11 18 38 AM](https://github.com/user-attachments/assets/32386d3c-237e-406b-b001-40022f4346d3)